File: SourceCode/API/educashAPI/Data/Entity/Preset.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace educashAPI.Data.Entity
{
    [Table("Preset")]
    public class Preset
    {
        [Key]
        public int PresetId { get; set; }
        [JsonIgnore]
        public int? userId { get; set; }
        public string PresetName { get; set; }
        public decimal Price { get; set; }
    }
}

File: SourceCode/andriod/app/src/main/java/com/example/educash/Preset.java
package com.example.educash;

public class Preset {
    private int presetId;
    private String presetName;

    private Double price;

    public Preset(int presetId, String presetName, Double price) {
        this.presetId = presetId;
        this.presetName = presetName;
        this.price = price;
    }

    public int getPresetId() {
        return presetId;
    }

    public String getPresetName() {
        return presetName;
    }

    public Double getPrice() { return price; }
}

